app/models/spree/calculator/avalara_transaction_calculator.rb

```diff
@@ -76,6 +76,11 @@ def tax_for_item(item, avalara_response)
       return 0 if item_address.nil?
       return 0 if !self.calculable.zone.include?(item_address)
 
+      if avalara_response['TaxLines'].blank?
+        Rails.logger.error "Avatax response property `TaxLines` null for order #{order.try(:number)} item ID #{item.try(:id)}."
```

Comment:
OK, I'll look up how to do this and then put up a PR with the new versioned branch.